### ISSUE TITLE
Support for comment package?

### ISSUE THREAD

**Issue by yxwangcs** [google-research/arxiv-latex-cleaner#6]
Thanks for this great project, it would nice to see support for `comment` package.

```
\begin{comment}
This can also be treated as comments and properly removed.
\end{comment}
```

**Comment by jponttuset** [comment]
Thanks RyanWangGit!
I am aware of this limitation, and I plan to add support for it at some point.

**Comment by RyanWangGit** [comment]
Great to hear that!

Just FYI the following regex seems to be working for me to capture comment (also works in nested case), not sure if it breaks anything, though.
```
\\begin\{comment\}[\s\S]*\\end{comment}
```

**Comment by jponttuset** [comment]
I implemented this in commit [367d60724a6b04467d4f85487b385edaa8baedf1](https://github.com/google-research/arxiv-latex-cleaner/commit/367d60724a6b04467d4f85487b385edaa8baedf1).
